frontend/packages/wizard/src/wizard.tsx

```diff
@@ -139,20 +149,31 @@ const Wizard = ({ heading, width = "default", dataLayout, children }: WizardProp
           </WizardContext.Provider>
         </DataLayoutContext.Provider>
         <Grid container justify="center">
-          {((state.activeStep === lastStepIndex && !isLoading && isMultistep) || hasError) && (
-            <ButtonGroup>
-              <Button
-                text="Start Over"
-                onClick={() => {
-                  dataLayoutManager.reset();
-                  setSearchParams({});
-                  dispatch(WizardAction.RESET);
-                  if (origin) {
-                    navigate(origin);
-                  }
-                }}
-              />
-            </ButtonGroup>
+          {state.activeStep === lastStepIndex && !isLoading && (
+            <>
+              {enableFeedback && (
+                <SimpleFeatureFlag feature="npsWizard">
+                  <FeatureOn>
+                    <NPSWizard />
+                  </FeatureOn>
+                </SimpleFeatureFlag>
+              )}
+              {(isMultistep || hasError) && (
```

Comment:
it looks like currently the `hasError` is outside of the `{state.activeStep === lastStepIndex` check https://github.com/lyft/clutch/blob/main/frontend/packages/wizard/src/wizard.tsx#L142 but here it's within

Reply:
Oops, missed the `||` statement being at the top level. Adjusting!